### DISCUSSION TITLE
Does this also work with the Bluetooth Adapter Bed?

### DISCUSSION THREAD

**Discussion by Fischkopppp** [krzmaz/LinakDeskEsp32Controller#19] [General]
Great work! Does anyone tried this with the Bluetooth Adapter Bed from linak?
https://www.linak.com/products/accessories/bluetooth-adapter-bed/
Thank you :)

**Comment by krzmaz** [comment]
Hi, I haven't tried it and have no way of testing it, but If you have the Linak Bed Control app on an android phone, you could [record bluetooth logs](https://source.android.com/docs/core/connect/bluetooth/verifying_debugging#debugging-with-logs) and check the communication between the app and the bed. If you see similar communication to that of a desk, it should be fairly easy to modify this project for your needs
